apps/tai/lib/tai/venue_adapters/bitmex/stream/trades.ex

```diff
@@ -5,7 +5,8 @@ defmodule Tai.VenueAdapters.Bitmex.Stream.Trades do
           "timestamp" => timestamp,
           "price" => price,
           "size" => qty,
-          "side" => side
+          "side" => side,
+          "trdMatchID" => client_id
```

Comment:
Is this `Tai.Trading.Order#client_id`?

Reply:
It could be renamed to `trade_id` or `remote_id` to be more generic instead of `client_id`?

Reply:
How about `venue_trade_id`? Similar to [`Order`](https://github.com/fremantle-capital/tai/blob/master/apps/tai/lib/tai/trading/order.ex#L32)